scripts/deploy_mcp_server.sh

```
@@ -0,0 +1,219 @@
+#!/bin/bash
+# Deploy MCP Server to AWS EC2
+set -e
+
+echo "🚀 MCP Server AWS Deployment Script"
+echo "===================================="
+
+# Configuration
+GITHUB_APP_ID="1554609"
```

Shellcheck (0.10.0)

[warning] GITHUB_APP_ID appears unused. Verify use (or export if used externally).

(SC2034)